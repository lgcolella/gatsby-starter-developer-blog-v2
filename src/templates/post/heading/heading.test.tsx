/* Vendor imports */
import React from 'react'
import { shallow } from 'enzyme'
/* App imports */
import Heading from './index'
import { ImageSharpFluid } from '../../../../tests/mocks'

describe('Post > Heading component', () => {
  test('matches snapshot', () => {
    let wrapper = shallow(
      <Heading
        title={'Title of the post'}
        tags={['angular', 'electron', 'nodejs']}
        cover={ImageSharpFluid['fluid']}
        coverTitle={'Title of the cover'}
      />
    )

    expect(wrapper).toMatchSnapshot()
  })
})
