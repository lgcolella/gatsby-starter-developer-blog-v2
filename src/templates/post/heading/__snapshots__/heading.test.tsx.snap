// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post > Heading component matches snapshot 1`] = `
<div
  className="container"
>
  <Image
    Tag="div"
    alt="Title of the post"
    durationFadeIn={500}
    fadeIn={true}
    fluid={
      Object {
        "aspectRatio": 1.7,
        "sizes": "(max-width: 600px) 100vw, 600px",
        "src": "/static/da8d0fc0af7f0d310517048e303b3fcf/a5a41/preview.jpg",
        "srcSet": "/static/da8d0fc0af7f0d310517048e303b3fcf/a5a41/preview.jpg",
      }
    }
    loading="lazy"
    title="Title of the cover"
  />
  <div
    className="title"
  >
    <h1>
      Title of the post
    </h1>
    <TagList
      tags={
        Array [
          "angular",
          "electron",
          "nodejs",
        ]
      }
    />
  </div>
</div>
`;
